### PR TITLE
fix incorrect parameter key in installApp (fix appium/appium#6695)

### PR DESCRIPTION
**Pull request by jlipps**
cc @imurchie

### CODE REVIEW

File: lib/commands/general.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ commands.installApp = async function (appPath) {
 
   let {apkPackage} = await this.adb.packageAndLaunchActivityFromManifest(appPath);
   let opts = {
-    localApkPath: appPath,
+    app: appPath,
     pkg: apkPackage,
```
Comment (imurchie) [inline lib/commands/general.js]: This looks like it is looking for `appPackage`, not `pkg`.

https://github.com/appium/appium-android-driver/blob/c5c8c1bb0b855283c83c760e41a91ca03b003204/lib/android-helpers.js#L261

Comment (jlipps) [inline lib/commands/general.js]: right you are! fixed

**Comment by imurchie** [comment]
If you rebase with master the CI troubles should be gone.

**Comment by imurchie** [comment]
Published in `appium-android-driver@1.10.31`.

**Pull request merged**